README.md

```diff
@@ -3,6 +3,34 @@ Fast, adaptive, secure, and timeless design network architecture.
 
 A design architecture for the web that enables real-time updating of user experiences based on changing inputs.
 
+## Packages
+### FAST animation library (Beta)
+An animation library that simplifies interactions and animations using the Web Animation API. Provides a convenient abstraction layer over the Web Animation API while still allowing full access to it.
+
+### FAST browser extension (Beta)
+An extension for testing real-world localization, theming, and other production type scenarios in web development. Originally devised for Fluent on Web as a way to visualize prototypes in many different user scenarios. For example, view a website in one of the 194 different localized settings.
+
+### FAST React components base
+A set of generic base web components written in React.
```

Comment:
How about? "Base components written in React that deliver quality HTML markup optimized for performance, accessibility, and modern browsers. We've found 90% of the effort for all component frameworks often goes into just this. So we've believe delivering this as a package, then allowing others to create their design language on top of this can short circuit the design process."

Reply:
Sounds good - Just making sure we want to go with a more personalized voicing now?

Reply:
You can change the "voice" to our standard.  I forgot to scrub for that.  Good catch.

Reply:
Will do